src/v3/src/transformer/identify/transformIdentify.ts

```diff
@@ -36,8 +41,10 @@ export const transformIdentify: IdxStepTransformer = ({ formBag, widgetProps, tr
     // TODO: OKTA-508744 - to use rememberMe in features once Default values are added widgetProps.
     // (i.e. rememberMe is default = true in v2)
     } else if (features?.rememberMe !== false && features?.rememberMyUsernameOnOIE) {
-      const usernameCookie = getUsernameCookie();
-      data.identifier = usernameCookie;
+      if (!isClientTransaction) {
```

Comment:
Will probably want this condition to wrap the `if` condition above as well, the reason is because if the username is set in widget props, they will experience the same issue as the username being pulled from the cookie. 

i.e. 

```
if (!isClientTransaction) {
   if (username) {
      ...
   } else if (features?.rememberMe ... ) {
      ...
   }
}
```